### ISSUE TITLE
HM: Zooming in a small graph cuts its vetically to the original height

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2540]
A bit hard to describe. 
- You start with a graph that does not take the whole width of the empty space below the toolbar. 
- the result is that the matrix does not take up the whole height of the available space (it does take up the whole width, though).
- when you zoom in, so that the matrix gets larger, it gets cut off at its original bottom boundary

**Comment by urapadmin** [comment]
### plugin 0.6
- the solution here is that the matrix takes up the whole available space no matter how small its contents are
- when zooming in it will still grow within its original boundaries but those are equal with the Browser's boundaries
- Don't resize your browser's windows size. That will screw this up (the matrix will not get resized). That is complicated and I don't want to spend too much time on this.

**Comment by luizaogs** [comment]
I'm not sure I understand the original issue. In MKAP AA does not take up the whole width, but it does take up the whole height? 

<img width="1512" alt="Screenshot 2024-02-08 at 11 06 58 AM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/941c3802-9fb9-4625-8cc2-4192369a354f">

**Comment by luizaogs** [comment]
This is what happens when zooming in currently. Is that what was intended? 

<img width="1512" alt="Screenshot 2024-02-08 at 11 08 02 AM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/1849d659-0b42-463e-be02-de498bffc3d2">

**Comment by urapadmin** [comment]
ja, it is hard to explain. This is about the height, not about the width. I should have made a screenshot.
It can only be tested with matrices that are smaller (less high) than the available empty space to begin with. So AA is too tall as it uses the whole available height from the start.
CE in URAP should work, but most of PVD is like that, too. 
And then when you would have zoomed before I fixed this the original lower edge of the matrix would keep being the edge an in place and cut everything off that tries to go beyond.

**Comment by luizaogs** [comment]
So in other words, the above looks good yes? 

<img width="1512" alt="Screenshot 2024-02-08 at 12 24 45 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/84b95a2e-2a17-4e9f-bcb3-73011b4ce20c">

<img width="1512" alt="Screenshot 2024-02-08 at 12 24 50 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/b463406d-554b-47e6-aa2f-ff8d99ce8888">

**Comment by urapadmin** [comment]
yes, that looks fine
